src/worldgen/DensityFunction.ts

```diff
@@ -64,9 +64,9 @@ export namespace DensityFunction {
 		const root = Json.readObject(obj) ?? {}
 		const type = Json.readString(root.type)?.replace(/^minecraft:/, '')
 		switch (type) {
-			case 'blend_alpha': return Constant.ONE
-			case 'blend_offset': return Constant.ZERO
-			case 'beardifier': return Constant.ZERO
+			case 'blend_alpha': return new ConstantMinMax(1, 0, 1)
+			case 'blend_offset': return new ConstantMinMax(0, -Infinity, Infinity)
+			case 'beardifier': return new ConstantMinMax(0, -Infinity, Infinity)
```

Comment:
Even though these are constant for us, it makes sense to have the min and max values represent the ones in vanilla.
(on a second thought, maybe blend_alpha and blend_offset are only replaced if there is some blending to do?)